cmd/rb-main.go

```diff
@@ -20,6 +20,7 @@ package cmd
 import (
 	"context"
 	"fmt"
+	"github.com/minio/minio-go/v7"
```

Comment:
why have we moved this import statement?

Reply:
This was done automatically by go fmt tool